code/src/pages/MovieDetails.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+
+import "./moviedetails.css";
+
+export const MovieDetails = () => {
+  const { movie_id } = useParams();
+  const [movieDetails, setMovieDetails] = useState({});
+
+  useEffect(() => {
+    fetch(
+      `https://api.themoviedb.org/3/movie/${movie_id}?api_key=d54701f7b694b0188ac7819c5b24df16&language=en-US`
+    )
+      .then((response) => response.json())
+      .then((json) => setMovieDetails(json))
+      .catch((err) => console.error(err));
+  }, [movie_id]);
+
+  return (
+    <>
+      <div className="main-wrapper">
+        <div
+          className="the-background"
+          style={{
+            backgroundImage: `linear-gradient(rgba(0, 0, 0, 0) 70%, rgb(0, 0, 0) 100%), URL(https://image.tmdb.org/t/p/w1280${movieDetails.backdrop_path})`,
+          }}
+          alt={movieDetails.title}
+        >
+          <a href="/" className="button-back">
```

Comment:
you should use `<Link to="/">` component provided by the react router. `<a>` restarts your app that should not be happening.